text_extensions_for_pandas/io/conll.py

```diff
@@ -357,11 +367,11 @@ def _doc_to_df(doc: List[_SentenceData],
      the returned dataframe will contain *two* columns, holding IOB2 tags and
      entity type tags, respectively. For example, an input column "ent" will turn into
      output columns "ent_iob" and "ent_type".
-
     :param space_before_punct: If `True`, add whitespace before
      punctuation characters (and after left parentheses)
      when reconstructing the text of the document.
-    :return: DataFrame with four columns:
+    :param doc_line_nums: list of line numbers in parsed file for each token in the doc
```

Comment:
This additional argument can go away if line numbers are included in the `_SentenceData` data structure.